pkgs/applications/science/misc/simgrid/default.nix

```diff
@@ -14,18 +14,17 @@ with stdenv.lib;
 
 let
   optionOnOff = option: "${if option then "on" else "off"}";
-  urlVersion = replaceStrings ["."] ["_"];
 in
 
 stdenv.mkDerivation rec {
   name = "simgrid-${version}";
-  version = "3.17";
+  version = "3.19.1";
```

Comment:
version bump

Reply:
Yes, sorry for not mentioning that in the PR text it used to be more clear before I fixed the merge conflicts with master. I'll move this into it's own PR.